scripts/publish_main.py

```diff
@@ -0,0 +1,18 @@
+from ingestion.parsers.snapshot_download import get_snapshot_bydate, date_to_ISO
+from ingestion.parsers.publish_parser import populate_DB
+from ingestion.parsers.graph_generator import generate_graph
+import datetime
+from ingestion.database.publish_db_init import *
+
+if __name__ == "__main__":
+    LOCATION = 'debian_packages.db'
+    ARCH = "amd64"
+    date = datetime.date(2016, 12, 31)
+    end_date = datetime.date(2022, 12, 31)
```

Comment:
can we have `start` and `end` be parameters to be passed (e.g., through sys.argv or so) with the defaults that you set there ?